endpoint/setup.sh

```diff
@@ -14,5 +14,15 @@ route add -net 193.167.0.0 netmask 255.255.0.0 gw $GATEWAY
 # delete unused route
 route del -net $UNNEEDED_ROUTE netmask 255.255.255.0
 
+# XXX this relies on the IPv6 address being second in the "hostname -I" output
```

Comment:
Can you remove the `XXX` from the comment? It'd also be good to know if the output of `hostname` changed. Can you add an
```
echo "Endpoint's IPv6 address is $IP"
```
which can be helpful when it breaks? Same above for the v4 address, that also relies on the v4 address being the first in the output.